pom.xml

```diff
@@ -29,9 +30,9 @@
     <artifactId>forgerock-parent</artifactId>
     <version>2.0.7</version>
```

Comment:
i also had to make similar changes to existing versions in each of the base projects in order to get them to build, since each component needs a specific existing version of forgerock-parent, forgerock-json, etc, so in a way I've kind of already had to cross into the territory of modifying existing versions.

I'd say that as soon as we get into _new_ version numbers that don't match FR's versions, we should not allow any modification once we cut a release. but any changes we need to make right now to get things buildable with each version IMO should be acceptable.